libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -1403,17 +1405,17 @@ void IRGeneratorForStatements::endVisit(FunctionCall const& _functionCall)
 			<!saltSet>
 				let <address> := create(<value>, <memPos>, sub(<memEnd>, <memPos>))
 			</saltSet>
+			<releaseMemory>(<memPos>)
```

Comment:
This looks like a bugfix, doesn't it?

Shouldn't we handle this separately and/or add tests for this?

Reply:
`releaseMemory` is a no-op.